python/GafferUI/Menu.py

```diff
@@ -405,11 +408,31 @@ def __evaluateItemValue( self, itemValue ) :
 
 		return itemValue
 
+	@GafferUI.LazyMethod()
+	def _lazyRepolish(self):
+
+		self._repolish()
+
+	@GafferUI.LazyMethod()
+	def __lazyUpdateSearchMenu( self, text):
+
+		return self.__updateSearchMenu( str(text) )
+
+	def __doUpdateSearchMenu( self, text ) :
+
+		# do a lazyUpdate in case we delete characters
+		if self.__previousSearchText[:-1] == str(text):
+			self.__previousSearchText = str(text)
+			return self.__lazyUpdateSearchMenu( str(text) )
```

Comment:
LazyMethods don't return anything (because their execution is deferred), so the `return` here is misleading.